sdk/identity/azure-identity/azure/identity/_base.py

```diff
@@ -46,20 +46,23 @@ def __init__(self, tenant_id, client_id, secret, **kwargs):  # pylint:disable=un
 class CertificateCredentialBase(ABC):
     """Sans I/O base for certificate credentials"""
 
-    def __init__(self, tenant_id, client_id, certificate_path, **kwargs):  # pylint:disable=unused-argument
-        # type: (str, str, str, **Any) -> None
+    def __init__(self, tenant_id, client_id, certificate_path, password=None, **kwargs):  # pylint:disable=unused-argument
+        # type: (str, str, str, Optional[Union[str, bytes]], **Any) -> None
```

Comment:
Why positional parameter not kwargs?

Reply:
Thanks for catching this, I intended it as a keyword argument and documented it as such.